### PR TITLE
Who's Online shows indicators but not statistics

### PR DESCRIPTION
**Pull request by scottcwilson**
Fixed to show counts as well as red-yellow-green traffic lights.

### CODE REVIEW

File: admin/includes/template/dashboardWidgets/tplWhosOnline.php [hunk 1]
```diff
@@ -17,26 +17,31 @@
...
 <div class="row">
-  <div class="col-xs-6"><?php echo WO_TOTAL; ?> <?php echo $tplVars['widget']['total']; ?></div>
-  <div class="col-xs-6"><?php echo '<a href="' . zen_href_link(FILENAME_WHOS_ONLINE) . '">' . WO_FULL_DETAILS . '</a>'; ?></div>
+  <div class="col-xs-12">&nbsp;</div>
+</div>
+<div class="row">
+  <div class="col-xs-12"><?php echo WO_TOTAL; ?> <?php echo $widget['total']; ?></div>
+</div>
+<div class="row">
+  <div class="col-xs-12"><?php echo '<a href="' . zen_href_link(FILENAME_WHOS_ONLINE) . '">' . WO_FULL_DETAILS . '</a>'; ?></div>
```
Comment (drbyte) [inline admin/includes/template/dashboardWidgets/tplWhosOnline.php]: Why all the additional col-xs-12 rows being added here?

Comment (scottcwilson) [inline admin/includes/template/dashboardWidgets/tplWhosOnline.php]: I think it looks nicer if the summary stats are each on their own line.  If you disagree, I can back this out.

**Comment by drbyte** [comment]
I'm also trying to understand why you changed `$tplVars['widget']` to `$widget` ... when `$widget` is never defined ...

I already see counts in my whos-online widget. I'm not sure I understand what problem is being fixed here.

**Comment by drbyte** [comment]
> I already see counts in my whos-online widget. I'm not sure I understand what problem is being fixed here.

Oy ... well, I did before I did a fresh install. The new install has no counts even after starting some sessions in other windows/browsers.  Okay that's insanely weird.

**Comment by scottcwilson** [comment]
OK.  I have images of before and after if needed, but it sounds like you are convinced.

**Comment by drbyte** [comment]
Ya, definitely weird. Esp since all the counts were showing when I did the original PR for this widget too.

File: admin/includes/template/dashboardWidgets/tplWhosOnline.php (hunk 1 above)
Comment (drbyte) [inline admin/includes/template/dashboardWidgets/tplWhosOnline.php]: > I think it looks nicer if the summary stats are each on their own line. If you disagree, I can back this out.

I'm not bothered by adding the extra blank line before the totals.
But I think the "more details" link is confusing when it's right under the totals.  Maybe making the class be "col-xs-6 right" for the link might create the separation you're looking for?

**Comment by drbyte** [comment]
👍

**Comment by zcwilt** [comment]
:+1:

**Pull request merged**